code/modules/projectiles/guns/projectile/shotgun/pump.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/weapon/gun/projectile/shotgun/pump
-	name = "FS SG \"Kammerer\""
+	name = "shotgun pump"
```

Comment:
![emoji](https://user-images.githubusercontent.com/61743710/103411849-87b3e200-4b72-11eb-9ab3-4088fde48489.png)

Reply:
Cuz else it will force all subtypes to Frozen Star spawn tag